apis/Google.Cloud.Spanner.Data/docs/connection_string.md

```diff
@@ -64,4 +64,49 @@ v1.0.
 
 Example:
 
-- UseClrDefaultForNull=true
\ No newline at end of file
+- UseClrDefaultForNull=true
+
+# Timeout
+
+The default timeout used for `SpannerCommand.CommandTimeout`,
+`SpannerTransaction.CommitTimeout`, and other Spanner network
+operations. Defaults to 60 seconds.
+
+Example:
+
+- Timeout=30
+
+# AllowImmediateTimeouts
+
+When set to False, a timeout of 0 means an indefinite timeout.
```

Comment:
nit: false, true without the capital letter?

Reply:
It would typically be written as False (or True) in the connection string, and I believe that's how it would be formatted if you call ToString on the connection string too.

Reply:
OK